<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DevSetting_RT1052_Poe_60W_2_4</class>
 <widget class="QWidget" name="DevSetting_RT1052_Poe_60W_2_4">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AMP Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
     border: 1px solid #8f8f91;
     border-radius: 4px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }
QPushButton:checked{
	background-color: rgb(0, 178, 0);
}</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>A-In2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>280</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>A-Out1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>A-In1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AOut_Mixter">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>70</width>
     <height>111</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Mixer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>190</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>N-In1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>A-Out Mixer</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Input Control</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Gain</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="NIn1_Channel_Gain">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-60</number>
    </property>
    <property name="maximum">
     <number>12</number>
    </property>
   </widget>
   <widget class="QPushButton" name="AIn1_Channel_Mute">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>mute</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="NIn1_Channel_Eq">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Eq</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NIn1_Channel_Mute">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>mute</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="AIn1_Channel_Eq">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Eq</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="AIn1_Channel_Gain">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-60</number>
    </property>
    <property name="maximum">
     <number>12</number>
    </property>
   </widget>
   <widget class="QPushButton" name="AIn1_Channel_Dyn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>DYN</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>230</y>
     <width>501</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Output Control</string>
   </property>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delay</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="AIn1_Delay">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>46</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-60</number>
    </property>
    <property name="maximum">
     <number>12</number>
    </property>
   </widget>
   <widget class="QPushButton" name="AIn1_Phase">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Phase</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="AIn1_xover">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>xover</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="AIn1_Gain">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-60</number>
    </property>
    <property name="maximum">
     <number>12</number>
    </property>
   </widget>
   <widget class="QPushButton" name="AIn1_Eq">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Eq</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Gain</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="AIn1_Dyn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>DYN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AIn1_Mute">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>mute</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_delay_1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>239.021ms</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>430</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>N-Out1</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Mic backstoping Control</string>
   </property>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Gain</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="AIn2_Channel_Gain">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-60</number>
    </property>
    <property name="maximum">
     <number>12</number>
    </property>
   </widget>
   <widget class="QPushButton" name="AIn2_Channel_Mute">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>mute</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="AIn2_Channel_Eq">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Eq</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AIn2_xover">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>xover</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>151</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Input</string>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>33</width>
      <height>23</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>A2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="VolumeWidget" name="Meter_A_1" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>33</width>
      <height>121</height>
     </rect>
    </property>
    <property name="MaxValue" stdset="0">
     <number>0</number>
    </property>
    <property name="MinValue" stdset="0">
     <number>-70</number>
    </property>
    <property name="SingleTrack" stdset="0">
     <bool>true</bool>
    </property>
    <property name="ScaleLabel" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>33</width>
      <height>23</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>A1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="VolumeWidget" name="Meter_A_2" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>33</width>
      <height>121</height>
     </rect>
    </property>
    <property name="MaxValue" stdset="0">
     <number>0</number>
    </property>
    <property name="MinValue" stdset="0">
     <number>-70</number>
    </property>
    <property name="SingleTrack" stdset="0">
     <bool>true</bool>
    </property>
    <property name="ScaleLabel" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>33</width>
      <height>23</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>N1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="VolumeWidget" name="Meter_N_1" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>33</width>
      <height>121</height>
     </rect>
    </property>
    <property name="MaxValue" stdset="0">
     <number>0</number>
    </property>
    <property name="MinValue" stdset="0">
     <number>-70</number>
    </property>
    <property name="SingleTrack" stdset="0">
     <bool>true</bool>
    </property>
    <property name="ScaleLabel" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>151</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>33</width>
      <height>23</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>A2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>33</width>
      <height>23</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>A1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="VolumeWidget" name="Meter_AL" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>33</width>
      <height>121</height>
     </rect>
    </property>
    <property name="MaxValue" stdset="0">
     <number>0</number>
    </property>
    <property name="MinValue" stdset="0">
     <number>-70</number>
    </property>
    <property name="SingleTrack" stdset="0">
     <bool>true</bool>
    </property>
    <property name="ScaleLabel" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="VolumeWidget" name="Meter_AR" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>33</width>
      <height>121</height>
     </rect>
    </property>
    <property name="MaxValue" stdset="0">
     <number>0</number>
    </property>
    <property name="MinValue" stdset="0">
     <number>-70</number>
    </property>
    <property name="SingleTrack" stdset="0">
     <bool>true</bool>
    </property>
    <property name="ScaleLabel" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>33</width>
      <height>23</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>N1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="VolumeWidget" name="Meter_N_Out" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>33</width>
      <height>121</height>
     </rect>
    </property>
    <property name="MaxValue" stdset="0">
     <number>0</number>
    </property>
    <property name="MinValue" stdset="0">
     <number>-70</number>
    </property>
    <property name="SingleTrack" stdset="0">
     <bool>true</bool>
    </property>
    <property name="ScaleLabel" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>201</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Status</string>
   </property>
   <widget class="QLabel" name="_Temperature">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="_24v">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>24V</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VolumeWidget</class>
   <extends>QWidget</extends>
   <header>customWidget/volumewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
